src/Components/Server/src/DependencyInjection/ComponentDescriptor.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Components.Server
+{
+    internal class ComponentDescriptor
+    {
+        public Type ComponentType { get; set; }
+
+        public string Selector { get; internal set; }
+
+        public void Deconstruct(out Type componentType, out string domElementSelector)
```

Comment:
nit: match the property and parameter name

Reply:
done, again ctrl+.